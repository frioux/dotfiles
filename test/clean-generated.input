#!/bin/bash

dir='test/,clean-generated'
generated_file="$dir/generated-file"
static_file="$dir/static-file"

mkdir -p "$dir"
touch "$generated_file"
touch "$static_file"

echo 'BEFORE {'
find "$dir"
echo '}'

make --always-make >/dev/null \
  "INSTALLATION_DIR=$dir" \
  "TARGETS_GENERATED=$generated_file" \
  clean-generated

echo 'AFTER {'
find "$dir"
echo '}'

rm -rf "$dir" "$generated_file" "$static_file"

# __END__
