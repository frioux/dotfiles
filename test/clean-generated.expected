BEFORE {
test/,clean-generated
test/,clean-generated/generated-file
test/,clean-generated/static-file
}
AFTER {
test/,clean-generated
test/,clean-generated/static-file
}
